keyloop/ext/sqla/identity.py

```diff
@@ -57,14 +57,23 @@ def _set_password(value):
         return bcrypt.encode(value)
 
     @singletonmethod
-    def get(self, identity_id):
-        try:
-            return self.session \
-                .query(self.model) \
-                .filter(self.model.id == identity_id, self.model.active == True) \
-                .one()
-        except NoResultFound:
-            raise IdentityNotFound
+    def get_by(self, **kwargs):
+        if 'username' in kwargs.keys():
```

Comment:
Podemos usar com sem o .keys()
```suggestion
        if 'username' in kwargs:
```